cads_adaptors/adaptors/multi.py

```diff
@@ -120,11 +120,7 @@ def retrieve(self, request: Request):
                 )
 
         logger.debug(f"MultiMarsCdsAdaptor, mapped_requests: {mapped_requests}")
-        result, output = execute_mars(mapped_requests)
-
-        self.context.stdout = output.stdout.decode()
-        self.context.stderr = output.stderr.decode()
-        self.context.user_visible_log = output.stdout.decode()
+        result = execute_mars(mapped_requests, context=self.context)
```

Comment:
```suggestion
        result, _ = execute_mars(mapped_requests, context=self.context)
```

Reply:
Alternatively we revert execute_mars to only return the result as we no longer use the output... outside of execute_mars

Reply:
OK, I'll return result only. I wasn't sure if that function is used elsewhere and didn't want to introduce breaking changes.